.env.development

```diff
@@ -1,9 +1,6 @@
-NEXT_PUBLIC_MAILCHIMP_ID='39a8feead1'
-NEXT_PUBLIC_MAILCHIMP_SUBSCRIBE_FORM_U='cc391beefd07bdde0f03f8e95'
-NEXT_PUBLIC_MAILCHIMP_SUBSCRIBE_FORM_C='jQuery190022527047066005612_1646552810702'
-NEXT_PUBLIC_MAILCHIMP_SUBSCRIBE_FORM_='1646552810703'
 NEXT_PUBLIC_GITHUB_LINK='https://github.com/amplication/amplication'
 NEXT_PUBLIC_LINKEDIN_LINK='https://www.linkedin.com/company/amplication'
 NEXT_PUBLIC_TWITTER_LINK='https://twitter.com/amplication'
 NEXT_PUBLIC_SITE='https://amplication.com'
 NEXT_PUBLIC_MAIL='contact@amplication.com'
+NEXT_PRIVATE_HUBSPOT_API_KEY=''
```

Comment:
what is the value?

Reply:
This is just stubbed for the API key we need. I didn't want to save it here but we will need to make sure it is saved/used by the site.